src/quo2/components/drawers/drawer_buttons/style.cljs

```diff
@@ -26,18 +29,21 @@
    :padding-horizontal      20
    :border-top-left-radius  20
    :border-top-right-radius 20
+   :blur-amount             20
```

Comment:
hmm, afaik these are not styles rather props of the 'blur-view' component. It would be better imo to apply these in the corresponding view.cljs file